README.md

```diff
@@ -0,0 +1,62 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Create Enumerable
+
+> This project uses Ruby to create Enumerable.
+
+
+## Built With
+
+- Ruby
+- git
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+Clone the project by using terminal:
+
+```
+git clone https://github.com/adamteddychang/create-enumerable.git
+
+
+
+```
+
+
+
```

Comment:
- [ ] Please kindly delete white spaces in your `readme file` as this will make your `readme file` cleaner and more professional and will portray you as someone who pays attention to details.